### PR TITLE
Add schwimmbad

### PR DESCRIPTION
**Pull request by guillochon**

### CODE REVIEW

File: recipes/schwimmbad/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - schwimmbad
+
+about:
+  home: http://github.com/adrn/schwimmbad
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'A common interface for parallel processing pools'
+
+  # The remaining entries in this section are optional, but recommended
```
Comment (pkgw) [inline recipes/schwimmbad/meta.yaml]: You should delete this comment line.

**Comment by pkgw** [comment]
Should this work on Windows? The AppVeyor CI test wasn't even attempted. Sometimes the GitHub/CI hooks just fail randomly; closing and re-opening the PR usually gets them to be re-triggered.

File: recipes/schwimmbad/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    schwimmbad enables easy switching between serial execution, multiprocessing,
+    and MPI by providing a common interface.
+  doc_url: https://github.com/adrn/schwimmbad
+  dev_url: https://github.com/adrn/schwimmbad
+
+extra:
+  recipe-maintainers:
+    - guillochon
+    - pkgw
+    - adrn
```
Comment (synapticarbors) [inline recipes/schwimmbad/meta.yaml]: @adrn - are you ok being added as a co-maintainer of the `schwimmbad` recipe on conda-forge?

Comment (adrn) [inline recipes/schwimmbad/meta.yaml]: @synapticarbors Sure!

Comment (pkgw) [inline recipes/schwimmbad/meta.yaml]: Yeah, I've been coordinating with @guillochon on these things.

**Pull request merged**